infra/main.bicep

```diff
@@ -138,6 +141,52 @@ param azureOpenAIEmbeddingModel string = 'text-embedding-ada-002'
 @description('Azure OpenAI Embedding Model Name')
 param azureOpenAIEmbeddingModelName string = 'text-embedding-ada-002'
 
+@description('Azure OpenAI Embedding Model Capacity')
+param azureOpenAIEmbeddingModelCapacity int = 30
+
+@description('Name of Computer Vision Resource (if using GPT-4 and Vision Preview)')
+param computerVisionName string = 'computer-vision-${resourceToken}'
+
+@description('Name of Computer Vision Resource SKU (if using GPT-4 and Vision Preview)')
+@allowed([
+  'F0'
+  'S1'
+])
+param computerVisionSkuName string = 'S1'
+
+@description('Location of Computer Vision Resource (if using GPT-4 and Vision Preview)')
+@allowed([// List taken from https://azure.microsoft.com/en-us/pricing/details/cognitive-services/computer-vision/
```

Comment:
I got an error that my region wasn't supported when I used UK South

I think we should use these regions: https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/how-to/image-retrieval?tabs=csharp#:~:text=Be%20sure%20to%20create%20it%20in%20one%20of%20the%20permitted%20geographic%20regions%3A%20East%20US%2C%20France%20Central%2C%20Korea%20Central%2C%20North%20Europe%2C%20Southeast%20Asia%2C%20West%20Europe%2C%20West%20US.